### DISCUSSION TITLE
Cluster solution

### DISCUSSION THREAD

**Discussion by huongtx498** [Novage/wt-tracker#68] [General]
I want to scale mutil wt-tracker machine as a cluster, how can i do ?

**Comment by ElonJTrump** [comment]
PM2 can help you

**Comment by fremzuck** [comment]
> PM2 can help you

i try to use pm2 but peer drop.

**Comment by ElonJTrump** [comment]
Yes, I found the same result peers was less than single thread worker but I
can't say performance dropped. The best way to check is bandwidth ratio.

ในวันที่ อา. 27 พ.ย. 2022 19:50 fremzuck ***@***.***>
เขียนว่า:

**Comment by fremzuck** [comment]
> Yes, I found the same result peers was less than single thread worker but I can't say performance dropped. The best way to check is bandwidth ratio. ในวันที่ อา. 27 พ.ย. 2022 19:50 fremzuck ***@***.***> เขียนว่า:
> […](#)
> PM2 can help you i try to use pm2 but peer drop. — Reply to this email directly, view it on GitHub <[#51 (comment)](https://github.com/Novage/wt-tracker/issues/51#issuecomment-1328240062)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARB2534PDQTI5Q6GIL6JZHTWKNKHZANCNFSM5EYP7NHQ> . You are receiving this because you commented.Message ID: ***@***.***>

Will each thread have a separate peer ?
